### PR TITLE
disable construct intent for buggy zones

### PR DESCRIPTION
**Pull request by chrisfarms**
there's [a contract bug](https://github.com/playmint/ds/issues/1402) preventing the CONSTRUCT_BUILDING_MOBILE_UNIT action for working as intended when the zone id is 40 or more.

as a temporary measure we are disabling the build intent for the affected zones so that it reduces the potential confusion of either the resulting "out of bounds" error or the silent failing due to these errors only being debug visisble.

### CODE REVIEW

File: frontend/src/components/views/shell/index.tsx [hunk 1]
```diff
@@ -31,6 +31,12 @@ import styled from 'styled-components';
 import { pipe, subscribe } from 'wonka';
 import { styles } from './shell.styles';
 
+// there is a bug that means that the CONSTRUCT_BUILDING_MOBILE_UNIT action
+// fails for zone ids above 40. As a temporary measure to limit confusion we
+// will disable the construct button on these zones while we consider options
+// see: https://github.com/playmint/ds/issues/1402
+const isBuggyZone = (id: number) => id >= 40;
```
Comment (5p0rt5BEArD) [inline frontend/src/components/views/shell/index.tsx:38]: should it be >= 41?

Comment (chrisfarms) [reply frontend/src/components/views/shell/index.tsx:38]: was erring on the safe side of the off-by one ness

Comment (5p0rt5BEArD) [reply frontend/src/components/views/shell/index.tsx:38]: I'm going to test it

Comment (5p0rt5BEArD) [reply frontend/src/components/views/shell/index.tsx:38]: >= 41 is correct

I've confirmed the following two statements locally
- The bug is present on zone 41 but not on zone 40
- The button is not displayed for zone 41 but is displayed for zone 40 with `const isBuggyZone = (id: number) => id >= 41;`

**Review by 5p0rt5BEArD — APPROVED** [review]
👍

**Pull request merged**